aboutpage.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About us</title>
+</head>
+<style>
+   .box-1 {
+    width: 100%;
+    height: 100px;
+    background-color: grey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+ul {
+    list-style: none;
+    display: flex;
+    align-items: stretch;
+} 
+li {
+    margin:40px ;
+}
+.first-box {
+    width: 100%;
+    height: 60px;
+    background-color: whitesmoke;
+    display: flex;
+    justify-content: center;
+    justify-content: flex-end;
+}
+.firstbox2-text {
+    font-family: 'rokkit', serif;
+    display: flex;
+    align-items: center;
+    font-size: 20px; 
+}
+.firstbox2-text {
+    margin-right: 2%;
+}
+.firstbox1-text {
+    margin-right: 1%;
+}
+.box-3 {
+    width: 100%;
+    height: 600px;
+    background-color: #404040;
+    display: flex;
+    justify-content: space-between;
+}
+.box3-img {
+    border-radius: 20%;
+    margin-left: 13%;
+    margin-top: 4%;
+    height: 500px;
+}
+.box3-text {
+    border-radius: 5%;
+    margin-right: 2%;
+    margin-top: 8%;
+}
+p {
+    font-size: 19px;
+    font-family:cursive ;
+    width: 400px;
+    color: white;
+}
+h1{
+    color: white;
+    font-weight: bolder;
+}
+.last-box {
+    width: 100%;
+    height: 100px;
+    background-color: #cccccc;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+.box11-icon1 {
+    margin-left: 22px;
+}
+.box11 {
+    display: flex;
+    align-items: stretch;
+}
+.box11-icon2 {
+    margin: 3px;
+}
+.box11-icon3 {ss
+    margin: 3px;
+}
+.box11-icon4 {
+    margin: 3px;
+}
+.box11-first {
+    font-family: 'rokkit',serif;
+}
+.box11-last {
+    font-family: 'rokkit',serif;
+}
+</style>
+<body>
+    <section>
+    <div class="first-box">
+        <img class="firstbox-1"src="images/grid.svg"/>
+        <div class="firstbox1-text"><h3>About</h3></div>
+        <img class="firstbox-2" src="images/phone-call.svg"/>
```

Comment:
How about you do some readup on font-awesome? Better to use icons for things like this.